lectures/geom_series.md

```diff
@@ -920,7 +954,32 @@ plt.show()
 Increasing the marginal propensity to consume $b$ increases the
 path of output over time.
 
-Now we will compare the effects on output of increases in investment and government spending.
+```{solution-end}
+```
+
+
+```{exercise-start}
+:label: geom_ex3
+```
+Continue from {ref}`geom_ex1` with the following values
+
+```{code-cell} ipython3
+values = [0.3, 0.4]
+```
+
+First, plot the time paths of $y_t$ using the same initial values but with $i$ 
+taking the values above.
+
+Next, plot the time paths of $y_t$ with the same initial values but let $g$ take
+the values mentioned above.
```

Comment:
Since `values = [0.3, 0.4]` is just one line above this sentence, I reckon what `the values mentioned above` in this sentence refers to is getting clearer.